### PR TITLE
Update commands in Get Started guide to use `kubectl gs`

### PR DESCRIPTION
**Pull request by kuosandys**
Closes https://github.com/giantswarm/roadmap/issues/471.

This PR updates the instructions and commands in the Get Started guide to use `kubectl gs` for setting up a kubectl context for a workload cluster.

As the quick-start instructions for installing `kubectl gs` with Krew are the same across different platforms, the tabs for platform-specific instructions have been removed. Rather than repeating instructions here, the link to the `kubectl gs` installation guide is provided for alternative installation instructions without Krew.
Screen Shot 2021-11-09 at 18 59 34
Screen Shot 2021-11-09 at 19 17 40

### CODE REVIEW

File: src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx [hunk 1]
```diff
@@ -26,102 +27,74 @@ const GettingStartedGetAccess: React.FC<IGettingStartedGetAccessProps> = () => {
...
-          Next, we let <code>gsctl</code> do several things in one step:
+          To install without Krew, download the{' '}
+          <a
+            href='https://github.com/giantswarm/kubectl-gs/releases/latest'
+            target='_blank'
+            rel='noopener noreferrer'
+          >
+            latest release
+          </a>{' '}
+          from GitHub for your platform, unpack the binary, and move it to a
```
Comment (axbarsan) [inline src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx:75]: Not sure which wording is right here. When you're unpacking something, are you unpacking the box that it came in, or are you unpacking the contents of the box?

```suggestion
          from GitHub for your platform, unpack the archive, and move it to a
```

File: src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx [hunk 2]
```diff
@@ -141,32 +114,31 @@ const GettingStartedGetAccess: React.FC<IGettingStartedGetAccessProps> = () => {
...
           <Prompt>
             {`
-                gsctl --endpoint ${window.config.apiEndpoint} \\
-                  create kubeconfig \\
-                  --cluster ${clusterId} \\
-                  --certificate-organizations system:masters \\
-                  --auth-token ${loggedInUser.auth.token}`}
+                kubectl gs login ${k8sAPIUrl.current} \\
+                  --workload-cluster ${clusterId} \\
+                  --organization ${orgId} \\
```
Comment (axbarsan) [inline src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx:119]: This is no longer necessary 🥳 
```suggestion
```

File: src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx [hunk 3]
```diff
@@ -176,11 +148,11 @@ const GettingStartedGetAccess: React.FC<IGettingStartedGetAccessProps> = () => {
...
           />{' '}
-          <code>--certificate-organizations</code> is a flag that sets what
-          group you belong to when authenticating against the Kubernetes API.
-          The default superadmin group on RBAC (Role Based Access Control)
-          enabled clusters is <code>system:masters</code> . All clusters on AWS
-          have RBAC enabled, some of our on-prem (KVM) clusters do not.
+          <code>--certificate-group</code> is a flag that sets what group you
+          belong to when authenticating against the Kubernetes API. The default
+          superadmin group on RBAC (Role Based Access Control) enabled clusters
+          is <code>system:masters</code> . All clusters on AWS have RBAC
```
Comment (axbarsan) [inline src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx:154]:
```suggestion
          is <code>system:masters</code>. All clusters on AWS have RBAC
```

File: src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx [hunk 4]
```diff
@@ -176,11 +148,11 @@ const GettingStartedGetAccess: React.FC<IGettingStartedGetAccessProps> = () => {
...
+          enabled, some of our on-prem (KVM) clusters do not.
```
Comment (axbarsan) [inline src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx:155]: Not sure if it's worth mentioning `KVM` here, as KVM users won't see this page anyway

Comment (kuosandys) [reply src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx:155]: I'll remove the sentence 👍

**Review by axbarsan — APPROVED** [review]

File: src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx [hunk 5]
```diff
@@ -141,32 +114,30 @@ const GettingStartedGetAccess: React.FC<IGettingStartedGetAccessProps> = () => {
...
-            installation.
+            <code>--workload-cluster &lt;cluster_name&gt;</code> selects the
+            cluster to provide access to.
           </li>
           <li>
-            <code>--cluster &lt;cluster_id&gt;</code> selects the cluster to
-            provide access to.
+            <code>--organization &lt;organization&gt;</code> selects the
+            organization the cluster belongs to.
           </li>
```
Comment (axbarsan) [inline src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx:132]: We no longer need this part

**Pull request merged**